backend/src/main/resources/application-azure-test.yaml

```diff
@@ -0,0 +1,19 @@
+spring:
+  profiles.include: okta-dev, auth-dev
+graphql.servlet.cors:
+  allowed-origins:
+    - https://simple-report-api-test.azurewebsites.net
+    - https://simple-report-test.azureedge.net
+    - https://test.simplereport.gov
+  admin-emails:
```

Comment:
I sort of want to factor this out into a separate property file but I think the answer is to factor it out into Okta. 😁